dataland-frontend/src/frameworks/gdv/FrontendFrameworkDefinition.ts

```diff
@@ -0,0 +1,19 @@
+// <--WARNING--> THIS FILE IS AUTO-GENERATED BY THE FRAMEWORK-TOOLBOX AND WILL BE OVERWRITTEN
```

Comment:
Can someone explain me the new setup we have here with BaseFrameworkDefinition and FrontendFrameworkDefinition?

Reply:
Answered in one of the previous questions.